pkg/clean/clean_service.go

```diff
@@ -204,25 +215,30 @@ func (c *CleanupService) cleanupAutoCreatedVPCs(ctx context.Context) error {
 // cleanupVPCResources cleans up the VPCs and their children resources created by nsx-operator.
 func (c *CleanupService) cleanupVPCResources(ctx context.Context) error {
 	// Clean up the indirect VPC children resources before deleting the VPCs, otherwise, it may block VPC deletion request
+	resourceCount := c.getPreVPCDeletionResourceCount()
+	c.log.Info("Starting pre-VPC deletion cleanup", "resourceCount", resourceCount)
 	if err := c.cleanupBeforeVPCDeletion(ctx); err != nil {
-		c.log.Error(err, "Failed to clean up the resources before deleting VPCs")
+		c.log.Error(err, "Failed to clean up the resources before deleting VPCs", "resourceCount", resourceCount, "status", "failed")
 		return err
 	}
-	c.log.Info("Completed to clean up the resources before deleting VPCs")
+	c.log.Info("Completed to clean up the resources before deleting VPCs", "resourceCount", resourceCount, "status", "success")
 
```

Comment:
some of log using 'Completed to clean up',
some of log using 'Successfully deleted'